packages/connect/CHANGELOG.md

```diff
@@ -1,13 +1,15 @@
 |             Package              | Stable |    Canary    |
 | :------------------------------: | :----: | :----------: |
-|       npm @trezor/connect        | 9.4.7  | 9.4.8-beta.1 |
-|     npm @trezor/connect-web      | 9.4.7  | 9.4.8-beta.1 |
-| npm @trezor/connect-webextension | 9.4.7  | 9.4.8-beta.1 |
+|       npm @trezor/connect        | 9.4.7  | 9.4.8-beta.2 |
+|     npm @trezor/connect-web      | 9.4.7  | 9.4.8-beta.2 |
+| npm @trezor/connect-webextension | 9.4.7  | 9.4.8-beta.2 |
 |    npm @trezor/connect-mobile    |   -    | 0.0.1-beta.1 |
 
 |     Deployment     | Stable |    Canary    |
 | :----------------: | :----: | :----------: |
-| connect.trezor.io/ | 9.4.7  | 9.4.8-beta.1 |
+| connect.trezor.io/ | 9.4.7  | 9.4.8-beta.2 |
+
+| connect.trezor.io/ | 9.4.7 | 9.4.8-beta.1 |
```

Comment:
this line seems doubled

Reply:
Yes, I just fixed it manually. But will investigate why it happened. It did not happen in yesterday's release.